docs/administration/upgrade/v6.0_changes.adoc

```diff
@@ -65,82 +74,118 @@ NOTE: To continue to use an existing Red Hat managed Kibana deployment provided
+...
 ----
+Snippet highlights::
 
-Additionally, create the following *ClusterRoleBinding* if collecting audit logs:
+<1> apiVersion must now be *observability.openshift.io/v1*
+<2> kind is now the *ClusterLogForwarder* spec
+<3> spec *collector* now includes resources, nodeSelector and tolerations
+<4> default values shown for requests and limits
 
-----
-oc adm policy add-cluster-role-to-user collect-audit-logs -z logcollector
-----
+CPU and memory limits::
+As with all cluster resources, use the values shown above as a reference point and adjust as necessary. If your pipeline is complex, you may need more collector resources; if you have a more straightforward pipeline, you may need less.
+* `resources.limits` describes the maximum amount of compute resources allowed
 
-==== Management, Resource Allocation & Workload Scheduling
-Configuration of the management state (i.e. managed, unmanaged), resource request and limits, tolerations, and node selection
-are part of the new ClusterLogForwarder API.
+* `resources.requests` describes the minimum amount of compute resources required.  Defaults to values of *limits* if not specified
 
-.Previously part of the resource *ClusterLogging* (v5.9)
-[source, yaml]
-----
-apiVersion: logging.openshift.io/v1
-kind: ClusterLogging
-metadata:
-  name: instance
-spec:
-  managementState: Managed
-  collection:
-    resources:
-      limits: {}
-      requests: {}
-    nodeSelector: {}
-    tolerations: {}
-----
-.Now part of v6.0 *ClusterLogForwarder* along with inputs, outputs, filters and pipelines
+
+==== Permissions (Service Accounts)
+The *ClusterLogForwarder* now requires a service account, with correct RBAC permissions, to be specified in the configuration.
+
+.Required Service Account
 [source, yaml]
 ----
 apiVersion: observability.openshift.io/v1
 kind: ClusterLogForwarder
 metadata:
-  name: instance
+  name: my-forwarder
 spec:
-  managementState: Managed
-  collector:
-    resources:
-      limits: {}
-      requests: {}
-    nodeSelector: {}
-    tolerations: {}
+  serviceAccount:
+    name: logging-admin
 ...
 ----
+Administrators are required to *explicitly grant log collection permissions* to any service account referenced in the *ClusterLogForwarder*.   There are 3 cluster roles that can be bound to: *collect-application-logs*, *collect-infrastructure-logs* and *collect-audit-logs*.
 
-==== Inputs Specifications
+.ClusterRoleBinding for *logging-admin*
+----
+oc adm policy add-cluster-role-to-user collect-application-logs -z logging-admin
+oc adm policy add-cluster-role-to-user collect-infrastructure-logs -z logging-admin
+----
 
+Additionally, if collecting audit logs:
+----
+oc adm policy add-cluster-role-to-user collect-audit-logs -z logging-admin
+----
+
+.Legacy *openshift-logging*
+If your previous collectors are deployed in the namespace *openshift-logging* then you've likely been using the service account *logcollector* created by earlier versions of the operator.
```

Comment:
in the namespace... named 'instance' then you have been using 

This legacy case was the only one where we provided the SA for you.  All others from mCLF usage you still needed to provide the SA

Reply:
Oscar had basically the same feedback, and I did modify the section.   I don't think the details of this statement are actually that critical.   The main message we need to relay, is that you now need a SA with correct permissions.        It's possible we created it for you in the past, but either way... you need to take notice and ensure that you now need to create a SA with correct permissions.